File: src/components/Item.js
function Item(props){
    return (
        <div>
            <div classname="itemname">{props.name}</div>
            <div classname="itemPrice">{props.price}</div>
            <div classname="itemCategory">{props.category}</div>
        </div>
    )
    
}

export default Item;

File: Backend/src/main/java/com/example/newbackend/model/Item.java
package com.example.newbackend.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;
import org.springframework.stereotype.Component;

import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.Id;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Component

public class Item {

    @Id
    @GeneratedValue
    private Long id;
    private String name;
    private int price;
    private String category; //enum

    //generate, get-set
}
